app/src/main/java/org/kiwix/kiwixmobile/zim_manager/library_view/LibraryFragment.kt

```diff
@@ -142,6 +144,18 @@ class LibraryFragment : BaseFragment() {
     }
   }
 
+  private fun noInternetSnackbar() {
+    view?.snack(
+      R.string.no_network_connection,
+      R.string.settings,
+      { (::openNetworkSettings)(Intent(Settings.ACTION_WIFI_SETTINGS)) }
```

Comment:
what is happening here? you should be able to just say `::openNetworkSettings` without the `{}` or anything else.

Ahhhhh you are invoking a method reference and passing an intent as a parameter, yeah still no.